@here/harp-mapview/lib/image/ImageCache.ts

```diff
@@ -141,36 +106,13 @@ export class ImageCache {
     }
 
     /**
-     * Add an image definition, and optionally start loading the content.
-     *
-     * @param owner - {@link ImageCacheOwner} adding the image.
-     * @param url - URL of image.
-     * @param startLoading - Optional flag. If `true` the image will be loaded in the background.
-     * @param htmlElement - Optional containing a HtmlCanvasElement or a HtmlImageElement as
-     * content, if set, `startLoading = true` will start the rendering process in the background.
-     */
-    addImage(
-        owner: ImageCacheOwner,
-        url: string,
-        startLoading = true,
-        htmlElement?: HTMLImageElement | HTMLCanvasElement
-    ): ImageItem | Promise<ImageItem | undefined> | undefined {
-        const imageItem = this.registerImage(owner, url, undefined, htmlElement);
-        if (imageItem !== undefined && startLoading === true) {
-            return this.loadImage(imageItem);
-        }
-
-        return imageItem;
-    }
-
-    /**
-     * Remove an image from the cache.
+     * Remove an image from the cache..
      *
      * @param url - URL of the image.
-     * @param owner - Optional: Specify {@link ImageCacheOwner} removing the image.
+     * @param owner - Optional: Specify {@link any} removing the image.
      * @returns `true` if image has been removed.
      */
-    removeImage(url: string, owner?: ImageCacheOwner): boolean {
+    removeImage(url: string, owner?: any): boolean {
```

Comment:
An accidental removal without owner will remove it for all owners without their knowledge. I think owner is no longer optional.

Reply:
will do.